### PR TITLE
UIU-2727: BILL ACTUAL COST and DO NOT BILL increase code coverage

### PR DESCRIPTION
**Pull request by artem-blazhko**
## Purpose
Cover "Lost items requiring actual cost" functionality by tests.

## Refs
[UIU-2727](https://issues.folio.org/browse/UIU-2727)

### CODE REVIEW

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+const patronName = 'name';
+
+jest.mock('./components', () => ({
+  ...jest.requireActual('./components'),
+  ActualCostModal: jest.fn(() => null),
+  ActualCostConfirmModal: jest.fn(() => null),
+  InstanceDetails: jest.fn(() => null),
+}));
+jest.mock('./util', () => ({
+  getPatronName: jest.fn(() => patronName)
```
Comment (Dmitriy-Litvinenko) [inline src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js:45]: In test we should just check that getPatronName has been called with correct props. Could you please remove patronName?

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+    };
+
+    describe('Patron formatter', () => {
+      describe('when patron group is presented', () => {
+        beforeEach(() => {
+          render(basicListFormatter[COLUMNS_NAME.PATRON](basicActualCostRecord));
+        });
+
+        it('should render "patronName"', () => {
+          expect(screen.getByText(patronName)).toBeVisible();
```
Comment (Dmitriy-Litvinenko) [inline src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js:120]: Could you please check in test that getPatronName has been called with correct props?

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+        });
+
+        it('should render "patronGroup"', () => {
+          expect(screen.getByText(`(${basicActualCostRecord.user.patronGroup})`)).toBeVisible();
+        });
+      });
+
+      describe('when patron group is not presented', () => {
```
Comment (Dmitriy-Litvinenko) [inline src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js:128]: patronGroup this is required field for user creation. Could you please check necessity of this tests?

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostModal/ActualCostModal.test.js [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+        fireEvent.click(cancelButton);
+
+        expect(initialProps.setActualCostModal).toHaveBeenCalledWith(ACTUAL_COST_MODAL_DEFAULT);
+        expect(initialProps.setActualCost).toHaveBeenCalledWith(ACTUAL_COST_DEFAULT);
+      });
+    });
+  });
+
+  describe('when actual cost type equals "doNotBill"', () => {
```
Comment (Dmitriy-Litvinenko) [inline src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostModal/ActualCostModal.test.js:166]: Could you please check can we check that additionalInformationForPatron are absent?

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostConfirmModal/ActualCostConfirmModal.test.js [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+        expect(initialProps.setActualCostConfirmModal).toHaveBeenCalledWith(ACTUAL_COST_CONFIRM_MODAL_DEFAULT);
+        expect(initialProps.setActualCostModal).toHaveBeenCalledWith({
+          isOpen: true,
+        });
+      });
+    });
+  });
+
+  describe('when actual cost type equals "doNotBill"', () => {
```
Comment (Dmitriy-Litvinenko) [inline src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostConfirmModal/ActualCostConfirmModal.test.js:113]: Could you please check can we check that additionalInformationForPatron are absent

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.js [hunk 6]
```diff
@@ -87,6 +87,37 @@ export const triggerOnSort = (e, meta, onSort) => {
 
   return onSort(e, meta);
 };
+export const basicListFormatter = {
```
[the remark this answers is not in the document]
Comment (artem-blazhko) [reply src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.js:90]: Renamed

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js (hunk 1 above)
Comment (artem-blazhko) [reply src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js:45]: Done

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js (hunk 2 above)
Comment (artem-blazhko) [reply src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js:120]: Done

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js (hunk 3 above)
Comment (artem-blazhko) [reply src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js:128]: Removed

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostModal/ActualCostModal.test.js (hunk 4 above)
Comment (artem-blazhko) [reply src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostModal/ActualCostModal.test.js:166]: Added initial test

File: src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostConfirmModal/ActualCostConfirmModal.test.js (hunk 5 above)
Comment (artem-blazhko) [reply src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostConfirmModal/ActualCostConfirmModal.test.js:113]: Added initial test

**Review by mkuklis — APPROVED** [review]

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Pull request merged**